lib/cosmos/system/system.rb

```diff
@@ -513,21 +521,30 @@ def load_configuration(name = nil)
                   # Zip file configuration so unzip and reset configuration path
                   configuration = unzip(configuration)
                 end
+                
+                Logger.info "Switching to configuration: #{name}"
                 process_file(File.join(configuration, 'system.txt'), configuration)
                 load_packets(name)
               rescue Exception => error
                 # Failed to load - Restore initial
+                @config_blacklist[name] = true # Prevent wasting time trying to load the bad configuration again
```

Comment:
Nice optimization but this probably never happens right? Exception loading a configuration?

Reply:
It happened when I copied over a saved config from another project and it was missing some conversions from their main lib folder...  Slowed everything to a complete crawl continually trying to load a defective config.

Reply:
Notably for this case, using the main lib folder is becoming an anti-pattern